site/index.md

```diff
@@ -17,6 +17,7 @@ The purpose of this documentation is to enable you to quickly on-board with our
 
 |Version Number|Date|Details|
 |--------------|----|-------|
+| 1.10.0 | 25th January 2019 | Added optional `Terminal` param for hotel with parking requests |
```

Comment:
being reeeeeeeeally picky here but everything else in the changelog ends in a full stop.